### PR TITLE
Overhaul article

### PR DESCRIPTION
**Pull request by dtzar**

### CODE REVIEW

File: docs/nv2-sign-verify-aks.md [hunk 1]
```diff
@@ -191,227 +46,79 @@ In this step, Gatekeeper will be configured, enabling deployment policies.
...
-2. Clear the ACR repo
-
-    ```bash
-    az acr repository delete \
-        -n $ACR_NAME \
-        --repository $ACR_REPO -y
-    ```
-
-3. Clear up AKS resources, leaving AKS in place
+Clear up ratify and gatekeeper resources, leaving AKS in place
```
Comment (sajayantony) [inline docs/nv2-sign-verify-aks.md:116]: Currently this is missing pulling from ACR. I'm guessing you will add this and also leave the registry in place?

Comment (dtzar) [reply docs/nv2-sign-verify-aks.md:116]: Yes, the thinking is they will use the images they already pulled from the previous article.

File: docs/nv2-sign-verify-aks.md [hunk 2]
```diff
@@ -191,227 +46,79 @@ In this step, Gatekeeper will be configured, enabling deployment policies.
         --name-template=gatekeeper \
         --namespace gatekeeper-system --create-namespace \
         --set enableExternalData=true \
-        --set controllerManager.dnsPolicy=ClusterFirst,audit.dnsPolicy=ClusterFirst
+        --set controllerManager.dnsPolicy=ClusterFirst,audit.dnsPolicy=ClusterFirst \
```
Comment (sozercan) [inline docs/nv2-sign-verify-aks.md:49]: you don't need this line anymore

File: docs/nv2-sign-verify-aks.md [hunk 3]
```diff
@@ -191,227 +46,79 @@ In this step, Gatekeeper will be configured, enabling deployment policies.
...
-        --repository $ACR_REPO -y
-    ```
-
-3. Clear up AKS resources, leaving AKS in place
+Clear up ratify and gatekeeper resources, leaving AKS in place
 
     ```bash
     helm uninstall ratify
-    kubectl delete secret regcred
+    helm install gatekeeper/gatekeeper
```
Comment (sozercan) [inline docs/nv2-sign-verify-aks.md:120]: uninstall?

File: docs/nv2-sign-verify-aks.md [hunk 4]
```diff
@@ -191,227 +46,79 @@ In this step, Gatekeeper will be configured, enabling deployment policies.
...
     kubectl delete ns demo
-    kubectl delete ns freezone
+    kubectl delete K8sSignedImages ratify-constraint
```
Comment (sozercan) [inline docs/nv2-sign-verify-aks.md:122]: if you uninstall gk, this will cleanup automatically

**Pull request merged**